pkg/snapshot/store.go

```diff
@@ -798,15 +798,13 @@ func upsertBackupStorageLocation(ctx context.Context, bsl *velerov1.BackupStorag
 		return nil, errors.Wrap(err, "failed to create velero client")
 	}
 
-	if err := veleroClient.Update(ctx, bsl); err == nil {
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to update backup storage location")
-		}
+	err = veleroClient.Update(ctx, bsl)
+	if err == nil {
```

Comment:
could you change line 814 to `return nil, errors.Wrap(err, "failed to update backup storage location")`